core/src/main/java/bio/terra/pearl/core/service/export/formatters/module/KitRequestFormatter.java

```diff
@@ -43,4 +46,34 @@ public Map<String, String> toStringMap(EnrolleeExportData enrolleeData) {
         maxNumRepeats = Math.max(maxNumRepeats, sortedKitRequests.size());
         return allKitMap;
     }
+
+    public List<KitRequestDto> listFromStringMap(Map<String, String> enrolleeMap) {
+        List<KitRequestDto> kitRequests = new ArrayList<>();
+        for (int itr = 1; itr < 10; itr++) {
```

Comment:
Since we've got nested for loops, let's use an actual variable name to make it clearer what this is iterating on
```suggestion
        for (int requestNum = 1; requestNum < 10; requestNum++) {
```